Rakefile

```diff
@@ -18,7 +18,7 @@ require_relative 'tasks/assets'
 require_relative 'tasks/profiles'
 require_relative 'tasks/geo'
 
-Mongoid.load!('./config/mongoid.yml')
+Mongoid.load!('./config/mongoid.yml', :test)
```

Comment:
I'm afraid `bundle exec rake test` does not work either:

```
$ bundle exec rake test
rake aborted!
Mongoid::Errors::NoEnvironment: 
```

As far as I understand, the ENV variable missing is not `ENV['RACK_ENV'] = 'test'` but `ENV['MONGOID_ENV'] = 'test'`. However adding it in the second line of the file `test/test_helper.rb` does not work, neither with `bundle exec rake test` nor with a plain `rake test`.

I might miss something basic in the configuration...?